src/infrastructure/transaction/CreateTransactionFromDTO.ts

```diff
@@ -159,10 +160,10 @@ const CreateStandaloneTransactionFromDTO = (transactionDTO, transactionInfo): Tr
             transactionDTO.nonce,
             new MosaicId(transactionDTO.mosaicId),
             new MosaicProperties(
-                new UInt64(transactionDTO.properties[0].value),
-                (new UInt64(transactionDTO.properties[1].value)).compact(),
-                transactionDTO.properties.length === 3 &&  transactionDTO.properties[2].value ?
-                    new UInt64(transactionDTO.properties[2].value) : undefined,
+                new UInt64(transactionDTO.properties[MosaicPropertyIdEnum.MosaicFlags].value),
```

Comment:
rename `MosaicPropertyIdEnum` to `MosaicPropertyType`  ?

Reply:
Fixed